### PR TITLE
Only show Delegations and Debonding tabs in account details

### PR DESCRIPTION
**Pull request by lukaw3d**
Part of https://github.com/oasisprotocol/oasis-wallet-web/issues/276

|  |  |  |
| --- | --- | --- |
|     localhost_3000_account_oasis1qpw6nzr77u5nfucee5wjp544hzgmpjjj2gz5p6zq_active-delegations(extension375x600 + scrollbar)   |  localhost_3000_account_oasis1qpw6nzr77u5nfucee5wjp544hzgmpjjj2gz5p6zq_active-delegations(extension375x600 + scrollbar) (1)      |  localhost_3000_account_oasis1qpw6nzr77u5nfucee5wjp544hzgmpjjj2gz5p6zq_active-delegations(extension375x600 + scrollbar) (2)   |

### CODE REVIEW

File: src/app/pages/AccountPage/index.tsx [hunk 1]
```diff
@@ -145,39 +106,7 @@ export function AccountPage(props: AccountPageProps) {
...
-              counter={stake.debondingDelegations?.length}
-              label={
-                isMobile
-                  ? t('account.subnavigation.mobileDebondingDelegations', 'Debonding')
-                  : t('account.subnavigation.debondingDelegations', 'Debonding delegations')
-              }
-              route="debonding-delegations"
-            />
-          </Nav>
+          <Box margin={{ bottom: 'small' }}></Box>
```
Comment (lubej) [inline src/app/pages/AccountPage/index.tsx:109]:
```suggestion
          <Box margin={{ bottom: 'small' }} />
```

**Review by lubej — COMMENTED** [review]
Why isn't tab selected on the second screenshot?

**Comment by lukaw3d** [comment]
> Why isn't tab selected on the second screenshot?

That's a tougher longstanding issue 

ROSE Wallet tab:
- /account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km
- /account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km/active-delegations
- /account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km/debonding-delegations

Stake tab:
- /account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km/stake

So highlighting by prefix doesn't work (stake has the same prefix). We could modify the routes to have a common prefix `/account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km/account/*`, but may need redirects for users with bookmarks. Or we could hardcode `/ || active-delegations || debonding-delegations`. Or group them in react router and tag the group. Or highlight first tab if no other tab matches :shrug:

**Comment by lubej** [comment]
> So highlighting by prefix doesn't work (stake has the same prefix). We could modify the routes to have a common prefix `/account/oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km/account/*`, but may need redirects for users with bookmarks. Or we could hardcode `/ || active-delegations || debonding-delegations`. Or group them in react router and tag the group. Or highlight first tab if no other tab matches 🤷

I am all for hardcoding `/ || active-delegations || debonding-delegations`. Not a fan of highlighting the first tab, if no other tab matches. The thing is, UX is kinda weird, if nothing is selected - at least to me. But would you select "ROSE Wallet" or "Stake"? Because "Stake" tab makes more sense to me.

**Comment by lukaw3d** [comment]
Maybe Stake tab makes more sense. Then this pullrequest is completely wrong direction

**Comment by lukaw3d** [comment]
Replaced by https://github.com/oasisprotocol/oasis-wallet-web/pull/1985

**Pull request closed without merging**